packages/components-react/src/Card/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { Heading3, Paragraph } from '@utrecht/component-library-react';
+import clsx from 'clsx';
+import { HTMLAttributes, PropsWithChildren } from 'react';
+import '@frameless/components-css/card/index.scss';
+
+// ensures that this component is flexible, can accept standard HTML attributes, and conditionally applies styles based on the background prop.
+interface CardProps extends HTMLAttributes<HTMLElement> {
+  background?: boolean;
+}
+
+export const Card = ({ background, children, className, ...restProps }: PropsWithChildren<CardProps>) => (
+  <div {...restProps} className={clsx(['frameless-card'], background, ['frameless-card-background'], className)}>
```

Comment:
Ik denk dat deze notatie van className niet werkt. Heb je getest dat de classNames erop komen zoals je verwacht?
Ik zou eerder dit denken:

```suggestion
  <div {...restProps} className={clsx('frameless-card', background && 'frameless-card-background', className)}>
```